docker/package/tezos_setup_wizard.py

```diff
@@ -447,6 +476,8 @@ def run_setup(self):
                 continue
             executed = True
 
+        self.set_toggle_vote()
```

Comment:
In some cases, this wizard can be run while the baking instance is already running. E.g. user wants to update its liquidity toggle vote. In that case, the baker service needs to be restarted manually to propagate changes, because config changes take effect only when service restarts

Reply:
I guess it should be safe to restart the whole baking instance when the wizard finishes